docs/literate/src/files/getting_started.jl

```diff
@@ -0,0 +1,288 @@
+# - Click the `Raw` button on the right side of the webpage.
+# - Right-click on any place of the newly opened webpage and choose `Save as`.
+# - Choose a folder and erase `.txt` from the file name. Save the file.
+
+# ### Files editing
+
+# For example, we will change the initial condition for calculations that occur in
+# `elixir_euler_ec.jl`. In this example we consider the compressible Euler equations:
+# ```math
+# \frac{\partial}{\partial t}
+# \begin{pmatrix}
+# \rho \\ \rho v_1 \\ \rho v_2 \\ \rho e
+# \end{pmatrix}
+# +
+# \frac{\partial}{\partial x}
+# \begin{pmatrix}
+# \rho v_1 \\ \rho v_1^2 + p \\ \rho v_1 v_2 \\ (\rho e +p) v_1
+# \end{pmatrix}
+# +
+# \frac{\partial}{\partial y}
+# \begin{pmatrix}
+# \rho v_2 \\ \rho v_1 v_2 \\ \rho v_2^2 + p \\ (\rho e +p) v_2
+# \end{pmatrix}
+# =
+# \begin{pmatrix}
+# 0 \\ 0 \\ 0 \\ 0
+# \end{pmatrix}
+# ```
+# for an ideal gas with ratio of specific heats ``\gamma`` 
+# in two space dimensions.
+# Here, ``\rho`` is the density, ``v_1``, ``v_2`` the velocities, ``e`` the specific total
+# energy, and
+# ```math
+# p = (\gamma - 1) \left( \rho e - \frac{1}{2} \rho (v_1^2+v_2^2) \right)
+# ```
+# the pressure.
+# Initial condition consists of initial values for ``\rho``, ``\rho v_1``,
+# ``\rho v_2`` and ``\rho e``.
+# One of the common initial condition for compressible Euler equations is density wave.
+# Let's implement it.
+
+# - Open the downloaded file with notepad or any other text editor.
+# - And go to the 9th line with following code:
+#   ````
+#   initial_condition = initial_condition_weak_blast_wave
+#   ````
+#   Here [`initial_condition_weak_blast_wave`](@ref) is used.
+# - Comment out this line using # symbol:
+#   ````
+#   # initial_condition = initial_condition_weak_blast_wave
+#   ````
+# - Now you can create your own initial conditions. Write following code into a file after the
+#   commented line:
+
+    function initial_condition_density_waves(x,t,equations::CompressibleEulerEquations2D)
+      v1 = 0.1 # velocity along x-axis
+      v2 = 0.2 # velocity along y-axis
+      rho = 1.0 + 0.98 * sin(pi * (sum(x) - t * (v1 + v2))) # density wave profile
+      p = 20 # pressure
+      rho_e = p / (equations.gamma - 1) + 1/2 * rho * (v1^2 + v2^2)
+      return SVector(rho, rho*v1, rho*v2, rho_e)
+    end
+    initial_condition=initial_condition_density_waves
+
+# - Execute following code one more time, but instead of `path_to_file` paste the path to the
+#   `elixir_euler_ec.jl` file from the current folder.
+#   ````
+#   using Trixi
+#   trixi_include(path_to_file)
+#   using Plots
+#   plot(sol)
+#   ````
+# Then you will obtain new solution.
+
+trixi_include(@__MODULE__,joinpath(examples_dir(), "tree_2d_dgsem", "elixir_euler_ec.jl"), #hide #md
+  callbacks=CallbackSet(StepsizeCallback(cfl=1.0)), initial_condition=initial_condition) #hide #md
+plot(sol) #hide #md
```

Comment:
The solution looks very ugly because of the weird choices for the axis limits by Plots.jl:
<img width="664" alt="image" src="https://user-images.githubusercontent.com/3637659/235155989-8bd97586-7968-4cc8-9fe1-b4b75b03c2a3.png">

Can you try to fix them please? You might need to use the `PlotData2D` approach and do something like this
```julia
pd = PlotData2D(sol)
p1 = plot(pd["rho"])
p2 = plot(pd["v1"], clim=(0.05, 0.25))
p3 = plot(pd["v2"], clim=(0.05, 0.25))
p4 = plot(pd["p"], clim=(10, 30))
plot(p1, p2, p3, p4)
```
which should give you something like
<img width="588" alt="image" src="https://user-images.githubusercontent.com/3637659/235163764-5774665f-c276-4a7a-90b6-2765a3b748d8.png">